src/content/api/bank-accounts.mdx

```diff
@@ -0,0 +1,746 @@
+    </Property>
+  </Properties>
+
+  ### Errors
+  | Status |                    Code                     |                                   Description                                   |
+  | :----- | :------------------------------------------ | :------------------------------------------------------------------------------ |
+  | 403    | BANK_ACCOUNT_ALREADY_VERIFIED               | The bank account is already verified.                                           |
+  | 403    | VERIFICATION_CODE_INVALID                   | The verification code is incorrect.                                             |
+  | 403    | BANK_ACCOUNT_VERIFICATION_ATTEMPTS_EXCEEDED | The bank account's maximum failed verification attempts has been reached.       |
+  | 403    | ACCOUNT_MISMATCH                            | The top up / withdrawal and the bank account do not belong to the same account. |
+
+  <CodeGroup slot="code-examples">
+    ```bash
+    curl -X POST https://service.centrapay.com/api/bank-accounts/WRhAxxWpTKb5U7pXyxQjjY/verify \
+      -H "X-Api-Key: $api_key" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "verificationCode": "1111"
+      }'
+    ```
+
+    ```json
+    {
+      "verificationCode": "1111"
+    }
+    ```
+  </CodeGroup>
+
+</Endpoint>
+
+---
+
+<Endpoint
+  method="POST"
+  path="/api/bank-authorities/{bankAccountId}/verify"
+>
+  ## Verify Bank Authority <Badge type="deprecated"/>
+
+  If you're creating new interfaces, please work with [Verify Bank Account](#verify-bank-account).
+
+  ### Attributes
+  <Properties>
+    <Property name="verificationCode" type="string" required>
+    	The code on the user’s bank statement.
+    </Property>
+  </Properties>
+
+  ### Errors
+  | Status |                    Code                     |                                   Description                                   |
+  | :----- | :------------------------------------------ | :------------------------------------------------------------------------------ |
+  | 403    | BANK_ACCOUNT_ALREADY_VERIFIED               | The bank account is already verified.                                           |
+  | 403    | VERIFICATION_CODE_INVALID                   | The verification code is incorrect.                                             |
+  | 403    | BANK_ACCOUNT_VERIFICATION_ATTEMPTS_EXCEEDED | The bank account's maximum failed verification attempts has been reached.       |
+  | 403    | ACCOUNT_MISMATCH                            | The top up / withdrawal and the bank account do not belong to the same account. |
+
+  <CodeGroup slot="code-examples">
+    ```bash
+    curl -X POST https://service.centrapay.com/api/bank-authorities/WRhAxxWpTKb5U7pXyxQjjY/verify \
+      -H "X-Api-Key: $api_key" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "verificationCode": "1111"
+      }'
+    ```
+
+    ```json
+    {
+      "verificationCode": "1111"
+    }
+    ```
+
+    ```json
```

Comment:
@AndyClifford I think this is a mistake in the existing docs. If you look at the implementation of the endpoint it just returns the verification code. This block can be removed 🧹